### DISCUSSION TITLE
no matter change the Nome or not, still not working

### DISCUSSION THREAD

**Discussion by TMD233** [giswqs/geohey#11] [Q&A]
Run codespell --skip="*.csv,*.geojson,*.json,*.js,*.html,*cff,./.git" --ignore-words-list="aci,hist"
./stat_01.egg-info/SOURCES.txt:868: Nome ==> Gnome
Error: Process completed with exit code 65.

**Comment by giswqs** [comment]
try lower case `nome`

**Reply by TMD233** [reply]
![屏幕截图 2024-03-26 143730](https://github.com/giswqs/geohey/assets/158091311/9f1c6906-cb99-4b7a-94e6-e2f8d4daedd7)

 The new problem is when I change the file, I can not synchronize or commit

![屏图 2024-03-26 144213](https://github.com/giswqs/geohey/assets/158091311/a09fce1d-1104-40ce-9f24-18a842d502de)

**Reply by giswqs** [reply]
Clone the main branch from GitHub and start clean. You probably created a another and messed it up.

**Reply by TMD233** [reply]
what should I do after I download the main branch from GitHub, Now there is no egg-info file in my folder, but still not passed the doc with the same issue.
